data-prepper-plugins/s3-source/src/integrationTest/java/org/opensearch/dataprepper/plugins/source/ParquetRecordsGenerator.java

```diff
@@ -38,6 +38,8 @@ public class ParquetRecordsGenerator implements RecordsGenerator {
 
     private static final String QUERY_STATEMENT ="select * from s3Object limit %d";
 
+    private static final String PARQUET_FILE_RELATIVE_PATH = "/src/main/resources/IntegrationTest.parquet";
```

Comment:
Thanks for working on this fix @udaych20 . I see that you added a new constant, but it is not used anywhere. I think you will need to use this for it to have an affect.